slither/core/declarations/function.py

```diff
@@ -94,14 +94,11 @@ class FunctionType(Enum):
 
 def _filter_state_variables_written(expressions: List["Expression"]):
     ret = []
+
     for expression in expressions:
-        if isinstance(expression, Identifier):
-            ret.append(expression)
-        if isinstance(expression, UnaryOperation):
-            ret.append(expression.expression)
-        if isinstance(expression, MemberAccess):
+        if isinstance(expression, (Identifier, UnaryOperation, MemberAccess)):
             ret.append(expression.expression)
```

Comment:
For `Identifier`, before it was:
```
        if isinstance(expression, Identifier):
            ret.append(expression)
```
Now it is:
```
        if isinstance(expression, (Identifier, UnaryOperation, MemberAccess)):
            ret.append(expression.expression)
```
Idk if it matters but was thing purposeful?

Reply:
The idea was to reduce the number of `isinstance` calls from 3 to 1, and add the property on `Identifier`.